script/landlongnode.ks

```diff
@@ -35,3 +48,13 @@ print "T+" + round(missiontime) + " Descent orbit burn: " + round(vom) + ", dv:"
 set nd to node(time:seconds + etam, 0, 0, deltav).
 add nd.
 
+// adjust for the body we are orbiting to rotate under us whilst we cruise to the descending node.
+set shiptonodedegrees to 360 * etam / body:rotationperiod.
+set adjustforshiptonodeseconds to ops * shiptonodedegrees / 360.
+// adjust for the body we are orbiting to rotate during the descent to periapsis to land with "run landv(3)." after the node is created.
+set adjustfororbitdegrees to 360 * (nd:orbit:period / 2) / body:rotationperiod.	
+set adjustfororbitseconds to ops * adjustfororbitdegrees / 360.

```

Comment:
should now read
set adjustfororbitseconds to sin(90 - ABS(nd:ORBIT:INCLINATION)) * ops * adjustfororbitdegrees / 360.